### PR TITLE
APEX-1587 - db setup

### PR DESCRIPTION
**Pull request by BaDo2001**
- Setup drizzle
- Setup docker compose for local development
- Setup RDS deployment
- Setup local migrations
- Setup remote migrations

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Postgres settings, used for local development, could be anything as long as they are consistent
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_DB=
+DB_URL=
```
Comment (attilazsoltdanku) [inline .env:5]: I guess the DB_URL could be pre-specified.
```suggestion
POSTGRES_USER=
POSTGRES_PASSWORD=
POSTGRES_DB=
POSTGRES_HOST=localhost
POSTGRES_PORT=5432

DB_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
```

Comment (attilazsoltdanku) [reply .env:5]: Ohh I was wrong :"D
We cannot use .env varibale into another .env variable.

**Review by attilazsoltdanku — APPROVED** [review]

File: .env (hunk 1 above)
Comment (BaDo2001) [reply .env:5]: Y its not possible like this. Also the .env file is under VC and I didn't want CI deployment to be effected hence the blank values.

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -76,11 +76,22 @@ jobs:
       - name: Deploy stack
         run: pnpm run deploy --stage ${{ env.STAGE }}
 
-      - name: Extract api endpoint
+      - name: Extract stack outputs
         id: sst-output
         run: |
           URL=$(jq -r '.["${{env.STAGE}}-birthday-slack-bot-MyStack"].ApiEndpoint' .sst/outputs.json)
           echo "apiEndpoint=$URL" >> "$GITHUB_OUTPUT"
+          MIGRATION_FUNCTION=$(jq -r '.["${{env.STAGE}}-birthday-slack-bot-MyStack"].MigrationFunctionName' .sst/outputs.json)
```
Comment (debrecenid) [inline .github/workflows/pr.yaml:84]: This step is not needed, if you give Name prop to the new Function in mystack, then the sst wont prefix it with the stage, so the function will be just MigrateDb

File: scripts/migrateCi.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { InvokeCommand, LambdaClient } from "@aws-sdk/client-lambda";
+
+const client = new LambdaClient({ region: "eu-central-1" });
+
+const res = await client.send(
+  new InvokeCommand({
+    FunctionName: process.argv[2],
```
Comment (debrecenid) [inline scripts/migrateCi.ts:7]: No need to pass the name as args, the function name will be "MigrateDb"

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (BaDo2001) [reply .github/workflows/pr.yaml:84]: Y but then it won't be unique anymore. E.g for me it collided with @attilazsoltdanku -s version when tried to deploy earlier.

Comment (debrecenid) [reply .github/workflows/pr.yaml:84]: True but also thats gonna be a problem as well, if there is no prefix.

Comment (BaDo2001) [reply .github/workflows/pr.yaml:84]: No because SST autogenerates the name if left empty.

Comment (debrecenid) [reply .github/workflows/pr.yaml:84]: Yup but it wasnt empty before 😄

**Review by debrecenid — APPROVED** [review]

**Pull request merged**